serverlessworkflow/modules/ROOT/pages/core/understanding-workflow-error-handling.adoc

```diff
@@ -20,10 +22,51 @@ Note that error definition for a Serverless Workflow is optional. If error handl
 
 An link:{spec_doc_url}#error-definition[error definition] in Serverless Workflow is composed of a `name` and `code` parameters. The `name` is a short and natural language description of an error, such as `wrong parameter`. The `code` parameter helps the implementation to identify the error.
 
-In Kogito, the `code` parameter is mandatory, which consists of the fully qualified Java class name of an link:{java_exception_url}[exception type].
+In Kogito, the `code` parameter is mandatory, and the engine uses different strategies to map the provided value to an exception encountered at runtime:
+
+==== Fully Qualified Class Name (FQCN)
+
+Define the mapping as the fully qualified Java class name of an link:{java_exception_url}[exception type].
 
 To determine if an exception is handled by an error definition, Kogito first checks whether or not the type of the Java exception is an instance of the exception that is specified in the error definition. Otherwise, Kogito recursively applies the same check to any embedded exception within the original Java one, till the exception chain is exhausted.
 
+**Examples**:
+[options="header"]
+|=======================
+|Runtime exception|Error code definition
+|`java.lang.RuntimeException`|`java.lang.RuntimeException`
+|`java.lang.IllegalStateException`|`java.lang.RuntimeException`
+|=======================
+
+==== Error message
```

Comment:
is this intentionally skipping one level of headers? i.e. `=== Error message`

Reply:
I have changed the header level